libbeat/docs/newdashboards.asciidoc

```diff
@@ -145,3 +285,33 @@ The output directory where the dashboards and all dependencies will be saved. Th
 NOTE: To make it even easier for you to import and export the dashboards, we've also added support for running the
 following commands in the Beats repository: `make import-dashboards` and `make export-dashboards`.
 
+The output directory has the following structure:
+
+[source,shell]
+--------------
+output/
+    index-pattern/
+    dashboard/
+    visualization/
+    search/
+--------------
+
+[[archive-dashboards]]
+=== Archive your Own Kibana dashboards
```

Comment:
Change to "Archiving Your Own Kibana Dashboards" to be consistent with other heading styles. We might change our doc style so we don't use gerunds at some point, but for now, let's be consistent.